src/messages/models/Banner/validateOptions.js

```diff
@@ -13,6 +13,8 @@ const VALID_OPTIONS = {
     partnerAccount: [Types.STRING],
     _legacy: [Types.BOOLEAN],
     onRender: [Types.FUNCTION],
+    onClick: [Types.FUNCTION],
+    onApply: [Types.FUNCTION],
```

Comment:
I think it would be great if function types always had a default empty function so that we wouldn't need to always surround them with `if` blocks where they are called. Let's add a `TODO:` comment to look into that later at some point unless you want to make that change now.

Reply:
I stuck in a TODO for now, but I'll keep plugging away at this.